### PR TITLE
chore(release): 5.1.0-beta.0

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Bump to 5.1.0-beta.0

# Changelog

- bumped versions to 5.1.0-beta.0
- updated release notes in `app-shell` and `api` with basic information 

# Review requests

-  release notes look reasonable for a first pass?
-  Changelogs look reasonable?

# Risk assessment

low
****

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -5,33 +5,51 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 [opentrons issue tracker]: https://github.com/Opentrons/opentrons/issues?q=is%3Aopen+is%3Aissue+label%3Abug
 
 ---
+# OT-2 Software Changes in 5.1.0-beta.0
 
-# OT-2 Software Changes in 5.0.2
+Welcome to the beta release of v5.0.0 release of the OT-2 software!
```
Comment (ecormany) [inline api/release-notes.md]:
```suggestion
Welcome to the beta release of v5.1.0 of the OT-2 software!
```

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -6,32 +6,61 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# Opentrons App Changes in 5.0.2
+# Opentrons App Changes in 5.1.0-beta.0
+Welcome to the beta release of v5.1.0 release of the Opentrons App!
```
Comment (ecormany) [inline app-shell/build/release-notes.md:11]:
```suggestion
Welcome to the beta release of v5.1.0 of the Opentrons App!
```

File: app-shell/build/release-notes.md [hunk 3]
```diff
@@ -6,32 +6,61 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+
+After you update your app, please be sure to update any OT-2's you will be using with this beta.
```
Comment (ecormany) [inline app-shell/build/release-notes.md:13]:
```suggestion
After you update your app, please be sure to update any OT-2s you will be using with this beta.
```
or "OT-2 robots". copyeditor's curse here 😆

**Review by ecormany — COMMENTED** [review]
Just a couple typo corrections for the beta readme files.

**Review by laviera — APPROVED** [review]

File: app-shell/build/release-notes.md [hunk 4]
```diff
@@ -6,32 +6,73 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# Opentrons App Changes in 5.0.2
+# Opentrons App Changes in 5.1.0-beta.0
+
+<<<<<<< Updated upstream
```
Comment (mcous) [inline app-shell/build/release-notes.md]: Looks like a merge conflict accidentally got committed!

**Review by mcous — APPROVED** [review]
LGTM! 📜

**Review by bndo — APPROVED** [review]
LGTM too 👍

**Pull request merged**